extensions/podman/packages/extension/src/extension.ts

```
@@ -1397,7 +1397,9 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
     const command = extensionApi.commands.registerCommand('podman.socketCompatibilityMode', async () => {
       // Manually check to see if the socket is disguised (this will be called when pressing the status bar item)
       const isDisguisedPodmanSocket = await isDisguisedPodman();
-
+      if (isDisguisedPodmanSocket === undefined) {
+        await extensionApi.window.showInformationMessage(`Could not get if Podman is disguised`);
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)